Copyedits for NHSHDx

davidmiller opened the pull request:
Pronouns, some tone and language tweaks.

I would kill the GDS strikethrough thing if we thought it was the wrong audience.

davidmiller committed:
Copyedits for NHSHDx

Pronouns, some tone and language tweaks

davidmiller committed:
Merge branch 'master' into davidmiller-patch-1

davidmiller commented:
P.S. Obviously I did all this in the browser without testing / checking, review welcome etc !

DeckOfPandas committed:
Tiny number of typo-like things

DeckOfPandas commented:
Comments:
* "a new evolution" is not a thing (sorry)
* You have "tightly focussed" in there twice
* There's a bonus </div> appearing in text at the bottom of the page

thebeckib committed:
Minor copy changes

Swapped out a few words
Took out the extra 'things we are trying to think about:' heading
